api/osb/check_visibility_plugin.go

```diff
@@ -15,6 +15,12 @@ import (
 
 const CheckVisibilityPluginName = "CheckVisibilityPlugin"
 
+const errPlanNotAccessible = &util.HTTPError{
+	ErrorType:   "ServicePlanNotAccessible",
+	Description: "service plan is not accessible",
+	StatusCode:  http.StatusForbidden,
```

Comment:
im not sure about this - for a client that has no visibility this practically means that he should probably not know that such plan exists

Reply:
We discussed this with @dpanayotov and decided to change the message to avoid confusion as clients usually do know that this plan exists. Sometimes they enable service access manually in CF and try to create an instance.

Reply:
Maybe it's better to say `service either not found or not accessible` with status 404? Isn't it an issue to directly say that there's such a plan but you don't have access?